build/testdata/bundles/mysql/porter.yaml

```diff
@@ -40,6 +44,8 @@ install:
     set:
       db.name: "{{ bundle.parameters.database-name}}"
       db.user: "{{ bundle.parameters.mysql-user }}"
+      primary.readinessProbe.timeout: "{{ bundle.parameters.probe-timeout}}"
```

Comment:
I added these parameters so that I can demonstrate overriding a dependency parameter in one of our tests without messing with one of the important connection string parameters.